cpp/ql/lib/semmle/code/cpp/PrintAST.qll

```diff
@@ -69,7 +71,7 @@ private predicate locationSortKeys(Locatable ast, string file, int line, int col
   )
 }
 
-private Function getEnclosingFunction(Locatable ast) {
+private Declaration getEnclosingDeclaration(Locatable ast) {
```

Comment:
The name of this predicate makes it sound like an `ast` should only have one enclosing declaration, but it looks like the predicate now can have multiple results when `ast` is an `Expr` or an `Initializer` (i.e., one result from `getEnclosingFunction` and another from `getEnclosingFunction`).

Functionally this is probably not an issue since it looks like we only use this predicate in the `shouldPrintDeclaration` filtering. But maybe we should rename this predicate to `getAnEnclosingDeclaration`?

Reply:
That makes sense. Will do.

Reply:
Done